apps/mymedicare_cb/views.py

```diff
@@ -72,28 +86,27 @@ def authenticate(request):
         headers.update({"X-Request-ID": str(getattr(request, '_logging_uuid', None)
                         if hasattr(request, '_logging_uuid') else '')})
 
-    response = requests.get(userinfo_endpoint,
-                            headers=headers,
-                            verify=sls_client.verify_ssl,
-                            hooks={
-                                'response': [
-                                    response_hook_wrapper(sender=SLSUserInfoResponse,
-                                                          auth_flow_dict=auth_flow_dict)]})
-
     try:
```

Comment:
After writing a test for this, found that the HTTPError exception wasn't being caught. So moved the TRY statement up before the requests.get().